facets_dive/components/facets_dive_vis/facets-dive-vis.ts

```diff
@@ -417,6 +417,11 @@ interface FacetingInfo {
 }
 
 /**
+ * A dictionary of booleans that is used to hold a set of data indices.
+ */
+export type IndexDict = {[index: string]: boolean};
```

Comment:
Should the index be a number?

Reply:
I removed this as I made the change simpler. Also don't need a custom event to fire as selected-indices-changed is already fired by polymer since selectedIndices has notify=true.